Flat/src/GeneralTree.cc

```diff
@@ -1001,21 +1033,23 @@ void GeneralTree::WriteTree(TTree *t) {
     Book("genAntiTopEta",&genAntiTopEta,"genAntiTopEta/F");
     Book("genTTPt",&genTTPt,"genTTPt/F");
     Book("genTTEta",&genTTEta,"genTTEta/F");
-    Book("jet1Flav",&jet1Flav,"jet1Flav/I");
-    Book("jet1Phi",&jet1Phi,"jet1Phi/F");
-    Book("jet1Pt",&jet1Pt,"jet1Pt/F");
-    Book("jet1GenPt",&jet1GenPt,"jet1GenPt/F");
-    Book("jet1Eta",&jet1Eta,"jet1Eta/F");
-    Book("jet1CSV",&jet1CSV,"jet1CSV/F");
-    Book("jet1CMVA",&jet1CMVA,"jet1CMVA/F");
-    Book("jet1IsTight",&jet1IsTight,"jet1IsTight/I");
-    Book("jet2Flav",&jet2Flav,"jet2Flav/I");
-    Book("jet2Phi",&jet2Phi,"jet2Phi/F");
-    Book("jet2Pt",&jet2Pt,"jet2Pt/F");
-    Book("jet2GenPt",&jet2GenPt,"jet2GenPt/F");
-    Book("jet2Eta",&jet2Eta,"jet2Eta/F");
-    Book("jet2CSV",&jet2CSV,"jet2CSV/F");
-    Book("jet2CMVA",&jet2CMVA,"jet2CMVA/F");
+    if (!leptonic) { // information about central jets is not needed for leptonic analyses
```

Comment:
Ditto

Reply:
I'll reply to you tomorrow. I'll see what I can change.

Reply:
Done